_posts/2023-09-21-hybrid-search.md

```diff
@@ -0,0 +1,389 @@
+---
+layout: post
+title:  Hybrid search is generally available in OpenSearch 2.10
+authors:
```

Comment:
I think we need to add Vamshi as well, @navneet1v what do you think?

Reply:
Yes lets add him too